charts/tfy-inferentia-operator/values.yaml

```diff
@@ -74,16 +93,16 @@ devicePlugin:
 scheduler:
   enabled: true
   schedulerName: neuron-scheduler
-  image: registry.k8s.io/kube-scheduler:v1.27.7
+  image:
+    repository: public.ecr.aws/eks-distro/kubernetes/kube-scheduler
+    tag: "v1.29.14-eks-1-29-latest"
```

Comment:
This is fine for now, later we will look into dynamically choosing based on k8s version

`Capabilities.KubeVersion.Major` and `Capabilities.KubeVersion.Minor`